docs/intro/tutorial01.txt

```diff
@@ -244,10 +157,6 @@ Now's a good time to note: **don't** use this server in anything resembling a
 production environment. It's intended only for use while developing. (We're in
 the business of making Web frameworks, not Web servers.)
 
-Now that the server's running, visit http://127.0.0.1:8000/ with your Web
```

Comment:
@dirtycoder I think we can now just restore this paragraph as it was before? That seems the right amount of mention for the welcome page. I don't think it needs a screenshot.